iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -682,6 +724,100 @@ static LogicalResult setRootConfig(
                                                tileSizes, passPipeline);
 }
 
+/// Sets the lowering configuration for a generic op implementing a
+/// transposition to use CPUDoubleTilingExpert pipeline.
+static LogicalResult setTransposeLikeOpRootConfig(
+    func::FuncOp entryPointFn, linalg::GenericOp genericOp,
+    ArrayRef<LoopTilingAndDistributionInfo> tiledLoops) {
+  if (getLoweringConfig(genericOp)) {
+    return success();
+  }
+
+  // If there are no loops, there is nothing to do.
+  unsigned numLoops = genericOp.getNumLoops();
+  if (numLoops == 0) {
+    return success();
+  }
```

Comment:
This should be part of `isSupportedTransposeOp` (and is already checked in the method).

Reply:
Good catch! Thanks!